lms/envs/common.py

```diff
@@ -2514,7 +2517,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-docstring
         'rest_framework.renderers.JSONRenderer',
     ),
     'PAGE_SIZE': 10,
-    'URL_FORMAT_OVERRIDE': None,
+#    'URL_FORMAT_OVERRIDE': None,
```

Comment:
Note to self: this has to get removed (move it to devstack? why do we have it here anyway?)